distro/src/app/components/HomePageHeader/styles/HomePageHeader.module.scss

```
@@ -9,18 +9,30 @@
 .header {
   display: flex;
   align-items: center;
-  padding: 0 $spacing-05;
+  width: 100%;
+  // Overrides Carbon's built-in header padding to accommodate the custom global bar layout
```

Stylelint (17.11.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)